src/libraries/System.Linq/tests/ConsistencyTests.cs

```diff
@@ -43,6 +43,7 @@ private static IEnumerable<string> GetExcludedMethods()
                 nameof(Enumerable.ToList),
                 nameof(Enumerable.ToHashSet),
                 nameof(Enumerable.TryGetNonEnumeratedCount),
+                nameof(Enumerable.Match),
```

Comment:
This would also need to be updated:
https://github.com/dotnet/runtime/blob/122c438e0987bf7bfd2a4a02368353d3774ff963/src/libraries/System.Linq.Queryable/tests/Queryable.cs#L86-L101